### ISSUE TITLE
[Feature Request] Dump kernel explorer output on crash

### ISSUE THREAD

**Issue by elad335** [RPCS3/rpcs3#9247]
Should output a string to log intead of GUI when a specific flag says so.

**Comment by elad335** [comment]
This actually may be better solved with a specialized savestate image which contains only LV2 stuff.

**Comment by digant73** [comment]
@elad335 maybe implemented in the last 5 years?

**Comment by elad335** [comment]
No. kernel explorer needs to be rewritten to be as bare as possible for this. We can't reuse Qt code for this. Qt needs to reuse the bare implementation.
